homeassistant/components/nws/weather.py

```diff
@@ -112,19 +108,19 @@ class NWSWeather(WeatherEntity):
     def __init__(
         self,
         entry_data: MappingProxyType[str, Any],
-        hass_data: dict[str, Any],
+        hass_data: NWSData,
```

Comment:
```suggestion
        nws_data: NWSData,
```
Maybe for the consistency